juneau-rest/juneau-rest-server/src/main/java/org/apache/juneau/rest/util/FinishableServletOutputStream.java

```diff
@@ -55,7 +55,6 @@ public final void flush() throws IOException {
 
 	@Override /* OutputStream */
 	public final void close() throws IOException {
-		os.close();
```

Comment:
Why was this removed?

Reply:
os is an OutputStream which has the same behaviour for the close method as the InputStream, as stated [here](https://docs.oracle.com/javase/8/docs/api/java/io/OutputStream.html#close--).

Reply:
Same reason as above.  The close() method on OutputStream may be a no-op, but it is not a no-op on subclasses.

Reply:
Done.